lib/dpl/provider/pages.rb

```diff
@@ -33,6 +34,8 @@ def initialize(context, options)
         @gh_email = options[:email] || 'deploy@travis-ci.org'
         @gh_name = "#{options[:name] || 'Deployment Bot'} (from Travis CI)"
 
+        @deployment_file = options[:deployment_file] && (options[:deployment_file]!=false)
```

Comment:
This is ugly. Try double negation, like `@deployment_file = !!options[:deployment_file]`